### PR TITLE
TST: Qtbot

### PR DESCRIPTION
**Pull request by teddyrendahl**
## Description

Add `qtbot` to all tests where widgets are created.

## Motivation and Context

I hate segfaults, you hate segfaults, we all hate segfaults

### CODE REVIEW

File: tests/test_log.py [hunk 1]
```diff
@@ -5,9 +5,10 @@
 from typhon.tools import TyphonLogDisplay
 
 
-def test_log_display(qapp):
+def test_log_display(qapp, qtbot):
```
Comment (hhslepicka) [inline tests/test_log.py]: `qapp` is not needed in here...

File: tests/test_plugins.py [hunk 2]
```diff
@@ -73,8 +75,9 @@ def test_metadata(qapp):
     assert widget._prec == 2
 
 
-def test_disconnection(qapp):
+def test_disconnection(qapp, qtbot):
```
Comment (hhslepicka) [inline tests/test_plugins.py]: `qapp` is not used in here...

Comment (teddyrendahl) [reply tests/test_plugins.py]: 👍

File: tests/test_log.py (hunk 1 above)
Comment (teddyrendahl) [reply tests/test_log.py]: 👍

**Comment by teddyrendahl** [comment]
@hhslepicka I tried to remove them and the tests failed. I'm guessing there is a better way to do it with `qtbot.waitUntil` but I kind of just want to move on.

**Comment by hhslepicka** [comment]
Agreed!

**Pull request merged**